annotate.el

```diff
@@ -756,12 +775,13 @@ to 'maximum-width'."
     (let ((newline-position (point)))
       (goto-char (1- (point))) ; we start at the start of the previous line
       ;; find overlays in the preceding line
-      (let ((prefix             (annotate-make-prefix)) ; white spaces before first annotation
-            (bol                (progn (beginning-of-line) (point)))
-            (eol                (progn (end-of-line) (point)))
-            (text               "")
-            (overlays           nil)
-            (annotation-counter 1))
+      (let* ((prefix-first       (annotate-make-prefix)) ; white spaces before first line of annotation
```

Comment:
If I'm reading this correctly, this doesn't need to be a `let*` any longer, and could be a `let` instead.

Reply:
Hi!

Yes there is no need for `let*` and moreover i think if we left it there this could be confusing for people that read the source, creating false expectations: that a previous binding is used in one of the next when truly is not.

Bye!
C.